src/openrct2/localisation/Language.h

```diff
@@ -1,5 +1,5 @@
 /*****************************************************************************
- * Copyright (c) 2014-2020 OpenRCT2 developers
+ * Copyright (c) 2014-2019 OpenRCT2 developers
```

Comment:
moved back in time?

Reply:
I have no idea what happened. I merged OpenRCT2 with my fork, pulled and this happened.

Reply:
You need to rebase to the latest `develop`. Do not merge our branches back into yours.